api/v1/validatorstate.go

```diff
@@ -94,8 +94,12 @@ func (v *ValidatorState) UnmarshalJSON(input []byte) error {
 	return err
 }
 
-func (v ValidatorState) String() string {
-	return validatorStateStrings[v]
+func (v ValidatorState) String() (string, error) {
```

Comment:
This changes the function signature so it no longer implements stringer.  The signature should stay just returning string, and return something like "unknown state" if v is out of bounds.

Reply:
yeah, good point. Updated to return `unknown` if v is out of bounds